src/components/Reports/ReportLayout/NavReportsList/NavReportListItem.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { ListItem, ListItemText } from '@material-ui/core';
+import { ArrowForwardIos } from '@material-ui/icons';
+import NextLink from 'next/link';
+import { useApp } from 'src/components/App';
+
+export interface ReportOption {
+  id: string;
+  title: string;
+  subTitle?: string | undefined;
```

Comment:
The `?` takes care of the undefined
```suggestion
  subTitle?: string;
```